### DISCUSSION TITLE
I can't enter your wallet (chia wallet)

### DISCUSSION THREAD

**Discussion by uguracikgozzz** [Chia-Network/chia-blockchain#4707] [General]
I logged out of the wallet for windows update. When I try to enter it again, I cannot enter. It says entering the wallet, but it does not.
![Ekran Görüntüsü (4)](https://user-images.githubusercontent.com/84088930/118002335-31408680-b350-11eb-8558-450879654559.png)

**Comment by erli655** [comment]
same Problem
